File: src/main/java/net/thedragonteam/thedragonlib/handlers/FileHandler.java
package net.thedragonteam.thedragonlib.handlers;

import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;

import java.io.File;

public class FileHandler {
    public static File configFolder;
    public static File mcDirectory;

    public static void init(FMLPreInitializationEvent event) {
        configFolder = event.getModConfigurationDirectory();
        mcDirectory = configFolder.getParentFile();
    }
}

File: src/main/java/net/thedragonteam/thedragonlib/handlers/FileHandler.kt
package net.thedragonteam.thedragonlib.handlers

import net.minecraftforge.fml.common.event.FMLPreInitializationEvent

import java.io.File

object FileHandler {
    lateinit var configFolder: File
    lateinit var mcDirectory: File

    fun init(event: FMLPreInitializationEvent) {
        configFolder = event.modConfigurationDirectory
        mcDirectory = configFolder.parentFile
    }
}